libmscore/sym.cpp

```diff
@@ -5321,6 +5321,7 @@ QVector<oldName> oldNames = {
       {"32' rest",                              SymId::rest32nd },            // rests.5
       {"64' rest",                              SymId::rest64th },            // rests.6
       {"128' rest",                             SymId::rest128th },           // rests.7
+      {"256' rest",                             SymId::rest256th },           // rests.8
```

Comment:
I do believe though that this PR is a too big and risky change for any branch of 2.0